spec/requests/rack_attack_spec.rb

```diff
@@ -1,14 +1,21 @@
 require "spec_helper"
 
 describe "Rack::Attack" do
+  around do |example|
+    default_store = Rack::Attack.cache.store
```

Comment:
Just to make sure I follow:

* we're remembering the default store is Rack::Attack.cache.store
* then we're overriding Rack::Attack.cache.store
* then we're running the example
* then we're setting Rack::Attack.cache.store back to its default state?

Reply:
Yeah. By default we connect to memcached:

https://github.com/otwcode/otwarchive/blob/700ccf065e91b4779370a183f6304bcb0780fd4c/config/environments/test.rb#L23-L24